chai.js

```diff
@@ -1847,16 +1847,59 @@ module.exports = function (chai, _) {
   Assertion.addMethod('haveOwnProperty', assertOwnProperty);
 
   /**
-   * ### .length(value)
+   * ### .ownPropertyDescriptor(name[, descriptor[, message]])
```

Comment:
The built source seems to create a funky diff — I haven't touched the `ownPropertyDescriptor` matcher at all.